tests/lib/cmsis_dsp/transform/testcase.yaml

```diff
@@ -70,7 +70,7 @@ tests:
       - native_sim
     tags: cmsis-dsp
     min_flash: 1024
-    min_ram: 64
+    min_ram: 71
```

Comment:
@EmilioCBen the 71 number is a bit arbitrary. Do you have data on what is actually needed by this test case?

Reply:
I tested on both the frdm_k64f and frdm_mcxw71 individually changing the bytes manually through a bisect to get this value.
The tests I changed all pass on RAM SIZE 72704 in Bytes and failed on 72703
expect for the case I mention on the description of this PR where the MCXW71 can pass the 
RF64 test with RAM SIZE 71680 exactly and not 71679.

Reply:
what if there is some platform that only has 64KB of ram physically but the software support is still able to fit in with this test < 64 KB? I don't think we should increase this, although I don't know how these min_ram are generally decided to begin with, it doesn't seem like a very precise requirement for any test given that there is so much variance in the actual sizes of images for the same test

Reply:
I am unsure of the answer to that.
I do not know what an alternative fix is for the testcase file.
I can remove this change but I would like a second confirmation
or a test coverage.